### PR TITLE
Adding use cases for distributed channels

### PR DESCRIPTION
**Pull request by strets123**
See also https://github.com/dask/distributed/pull/749

### CODE REVIEW

File: docs/source/use-cases.rst [hunk 1]
```diff
@@ -395,12 +395,22 @@ Streaming data engineering
...
-produce a rudimentary but effective stream processing system.
+produce a rudimentary but effective stream processing system. Alternatively, futures can be 
+streamed beween different clients using `channels`_ .
 
 Because dask.distributed is elastic, they can scale up or scale down their
 cluster resources in response to demand.
 
+End user application development
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dask and Distributed offer a 100% Python alternative to message queue based tools such as `Celery`_.
```
Comment (mrocklin) [inline docs/source/use-cases.rst]: 100% is a strong statement and I'm not sure that Dask can live up to it.  See http://matthewrocklin.com/blog/work/2016/09/13/dask-and-celery

File: docs/source/use-cases.rst [hunk 2]
```diff
@@ -395,12 +395,22 @@ Streaming data engineering
...
 
 Because dask.distributed is elastic, they can scale up or scale down their
 cluster resources in response to demand.
 
+End user application development
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dask and Distributed offer a 100% Python alternative to message queue based tools such as `Celery`_. 
+The developer who is looking to support power users for custom plugins but also basic users who need
+to run long running, pre-baked tasks can use distributed `channels`_ . This allows the distributed Client to 
+live only for the duration of a web request and the task status to be retrieved later or sent over a websocket.
```
Comment (mrocklin) [inline docs/source/use-cases.rst]: The title of this section seems very broad.  "End user application development" could mean a wealth of things from web applications, to games, to finance dashboards.  However the example given seems to be more about message queues for one-off tasks in a web server.  We should probably change the title.

**Comment by strets123** [comment]
I have tweaked the wording to talk about the strengths of message queue based tools without reference to pure python.

**Comment by mrocklin** [comment]
Channels have been removed

**Pull request closed without merging**